.Brewfile

```diff
@@ -35,6 +35,8 @@ brew 'gettext'
 # Magic
 brew 'go'
 brew 'jq'
+brew 'yarn'
+brew 'node@18'
```

Comment:
should we just install `node` instead and then install [nvm](https://github.com/nvm-sh/nvm) as well ?

```suggestion
brew 'node'
brew 'nvm'
```